src/app/notification-view-list/notification-view-list.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from '../shared/user.service';
+import { CouchService } from '../shared/couchdb.service';
+
+@Component({
+  template: `
+    <p i18n>Your Notifications</p>
+    <mat-list role="list" *ngFor="let notification of notifications">
+      <mat-list-item (click)="readNotification(notification)">
+      <mat-divider></mat-divider>
+        <p [ngClass]="{'menu-item-text':notification.status==='unread'}">
+          <a routerLink="{{notification.link}}">
```

Comment:
See my other question.  Right now the only notification link goes to this list, so this goes nowhere.  If we will have notifications lead to other pages, this is fine, but if not then this does not need to be a link.